Bottom: 0035e1711dadfbf0188fee14e6c41ca835f4f9f2
Top:    05618e3a5a906f54e1556fb9d13a52863d000514
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-09 17:07:06 -0500

Add parameter for buildbot version



---
diff --git a/docker/buildbot-master/Dockerfile b/docker/buildbot-master/Dockerfile
index 521c823..dde9cdb 100644
--- a/docker/buildbot-master/Dockerfile
+++ b/docker/buildbot-master/Dockerfile
@@ -1,4 +1,5 @@
-FROM        buildbot/buildbot-master:v0.9.13
+ARG         BUILDBOT_VERSION=latest
+FROM        buildbot/buildbot-master:$BUILDBOT_VERSION
 MAINTAINER  Andrew Cooper <andrew.cooper@novatechweb.com>
 
 WORKDIR /buildbot
